blueprints/ember-decorators/index.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  normalizeEntityName: function() {}, // no-op since we're just adding dependencies
+
+  afterInstall: function() {
+    return this.addPackagesToProject([
+      { name: 'babel-plugin-transform-class-properties' },
```

Comment:
Hmm.  Good point. How about we split the difference?  Basically we bring the decorators transform, and we keep the blueprint to add the `transform-class-properties` to the host app...